ci-automation/tapfile_helper_lib.sh

```
@@ -0,0 +1,281 @@
+#!/bin/bash
+#
```

There are some pipes used and I think we should do `set -euo pipefail` here instead of silencing the errors

---

Since these are pure library functions I'd leave this to the script sourcing the lib, but it's a good point. Will add it to `test.sh` and to `test_update_reruns.sh`.